test_files/type_alias_imported/type_alias_exporter.ts

```diff
@@ -0,0 +1,3 @@
+export class X { private x: string; }
+export class Y { private x: string; }
+export type XY = X|Y;
```

Comment:
Docs in here about what this is looking for are again helpful.

Reply:
Done.